website/content/docs/runner/index.mdx

```diff
@@ -35,7 +35,7 @@ uniquely configured environments.
 
 For that reason, Waypoint makes heavy usage of On-demand runners:
 
-### On-Demand Runners
+### On-demand runners
```

Comment:
```suggestion
### On-demand Runners
```

Should match the casing that the docs team uses, not sure if they have a copy style guide

Reply:
I'm making a separate PR for all the changes, so I'm going to "undo" this suggestion

Reply:
merged in https://github.com/hashicorp/waypoint/pull/4163